### PR TITLE
doc/md: terraform + mysql + rds guide

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/md/guides/mysql/terraform.md [hunk 1]
```diff
@@ -0,0 +1,434 @@
+---
+id: terraform-mysql-rds
+title: Manage MySQL on RDS with Terraform and Atlas
+slug: /guides/mysql/terraform
+tags: [terraform, rds, mysql, aws, iac]
+---
+
+### TL;DR
+
+* Many teams use [Terraform](https://terraform.io/), a popular, open-source, infrastructure-as-code
```
Comment (masseelch) [inline doc/md/guides/mysql/terraform.md:10]:
```suggestion
* Many teams use [Terraform](https://terraform.io/), a popular, open-source, infrastructure-as-code (IaC)
```

File: doc/md/guides/mysql/terraform.md [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+
+```text
+docker run --rm --name atlas-db-dev -d -p 3306:3306 -e MYSQL_DATABASE=example -e MYSQL_ROOT_PASSWORD=pass mysql:8
+```
+
+As reference for the next steps, the URL for the Dev Database will be:
+```text
+mysql://root:pass@localhost:3306/example
+```
+
```
Comment (masseelch) [inline doc/md/guides/mysql/terraform.md:62]:
```suggestion
```

File: doc/md/guides/mysql/terraform.md [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+  index "idx_email" {
+    columns = [
+      column.email
+    ]
+    unique = true
+  }
+  // highlight-end
+}
+```
+Now, run re-apply with Terraform:
```
Comment (masseelch) [inline doc/md/guides/mysql/terraform.md:368]:
```suggestion
Now, re-apply with Terraform:
```

**Review by masseelch — APPROVED** [review]
🚀

File: doc/md/guides/mysql/terraform.md [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+
+### Connect everything together
+
+Finally, configure the Atlas Terraform Provider to apply the schema in our `schema.hcl` file
+on the RDS-managed database instance. 
+
+```hcl
+// Load the schema from file and normalize it using the dev database.
+data "atlas_schema" "hello" {
+  dev_db_url = "mysql://root:pass@localhost:3306/example"
```
Comment (a8m) [inline doc/md/guides/mysql/terraform.md:222]: Define as `local` because it's nicer :)

**Review by a8m — APPROVED** [review]
🥳

**Pull request merged**